Jenkinsfile.pr.si

```diff
@@ -30,7 +30,7 @@ node('py36') {
             archive includes: "diagnostics.zip"
 
             withCredentials([usernamePassword(credentialsId:'a7ac7f84-64ea-4483-8e66-bb204484e58f',passwordVariable:'GIT_PASSWORD', usernameVariable: 'GIT_USER')]) {
-                sh """python3 ./ci/github_status.py "$JOB_NAME/${params.Variant}" "$JOB_URL" \$(git rev-parse HEAD) $currentBuild.result"""
+                sh """python3 ./ci/github_status.py "$JOB_NAME/${params.Variant}" "$BUILD_URL" \$(git rev-parse HEAD) $currentBuild.result"""
```

Comment:
don"t you wanna do the same thing also on line 24?

Reply:
You are right. Thanks for noticing.